src/game/game.h

```diff
@@ -59,6 +59,10 @@ struct enttypes
     bool    noisy,  syncs,  resyncs,    syncpos,    synckin;
     const char *name,           *attrs[13];
 };
+
+// delay in milliseconds between updates for client/server
```

Comment:
The description is not entirely accurate. It's the minimum interval for updates. All we guarantee is that updates are not sent more often than in this interval (i.e., we guarantee a lower boundary for this delay), but we cannot ensure updates are sent at this interval (e.g., with low frame rates, updates are sent much less often).
We just increases the *possible* update rate in this PR.